woocommerce/woocommerce-admin
# Add starter pack script

Adds a starter pack script for basic usage of [wp-scripts](https://github.com/WordPress/gutenberg/tree/master/packages/scripts) with modification for WooCommerce Admin. All linting and unit testing defaults to standard WP environment designated by wp-scripts.

### Detailed test instructions:

1. `npm run create-wc-extension`
2. `cd ..`
3. cd into your new folder (your chosen name lowercased and hyphenated)
4. `npm install && npm start`
4. Activate the plugin
5. Go to any wc-admin page and see "Hello World" appear in the console.

## Next Steps

* Possibly add our overrides such as `.eslintrc` or `phpunit.xml.dist`. We'd need to be sure these are the standards we'd like to promote to the ecosystem.
* If this is helpful, move this to `packages` so that developers can simply `npx create-wc-extension` independently of this repo.
